consensus/src/sync/history/cluster.rs

```diff
@@ -254,38 +237,21 @@ impl<TNetwork: Network + 'static> SyncCluster<TNetwork> {
             return Err(SyncClusterResult::Error);
         }
 
-        // Verify chunk.
-        let chunk = history_chunk.chunk.expect("History chunk missing");
-        if !chunk
-            .verify(
-                epoch.block.header.history_root.clone(),
-                chunk_index * CHUNK_SIZE,
-            )
-            .unwrap_or(false)
-        {
-            log::warn!(
-                "History Chunk failed to verify (chunk {} of epoch {})",
-                chunk_index,
-                epoch_number
-            );
-            return Err(SyncClusterResult::Error);
-        }
-
```

Comment:
Ok, here we use the same code as above. We should still test the code with more than 1 history chunk (tests seem to always only add 1).

Reply:
Added a new test: `history_sync_works_with_multiple_chunks`